scripts/gen_kobject_list.py

```diff
@@ -441,7 +441,7 @@ def find_kobjects(elf, syms):
                 continue
 
             loc = die.attributes["DW_AT_location"]
-            if loc.form != "DW_FORM_exprloc":
+            if loc.form != "DW_FORM_exprloc" and loc.form != "DW_FORM_block1":
```

Comment:
The DW_FORM_block1 might still be used here for compatibility with MetaWare - I'll check with the GNU team.

Reply:
I think this change should be moved to another patch which is concerned with enabling kernel objects, since everything else here is about enabling the MPU/memory domains.

do DW_FORM_exprloc and DW_FORM_block1 have the same format? subsequent code attempts to parse the data within it to get a memory address.

Reply:
It needs the check from ARC GNU team. From the the mem_domain_api test, it looks no problem. I will make the change in a separate commit